v3.1/reference/transactions/overview.md

```diff
@@ -5,9 +5,9 @@ category: reference
 
 # TiDB 事务概览
 
-TiDB 支持完整的分布式事务。本文主要介绍涉及到事务的语句、显式/隐式事务以及事务的隔离级别和惰性检查。
+TiDB 支持完整的分布式事务，默认使用[乐观事务模型](/v3.1/reference/transactions/transaction-optimistic/)。事务提交时假定不会发生并发冲突，只有在事务最终提交时才会检测冲突。同时，TiDB V3.0 版本也加入了[悲观事务](v3.1/reference/transactions/transaction-pessimistic.md)的支持。本文主要介绍涉及到事务的语句、显式/隐式事务以及事务的隔离级别和惰性检查，以及事务大小的限制。
```

Comment:
@anotherrachel Please confirm this description about default transaction model with @AndreMouche, to check whether the optimistic model is the default one in all versions.